dbt/adapters/spark/connections.py

```diff
@@ -507,56 +485,50 @@ def open(cls, connection):
         return connection
 
 
-def build_ssl_transport(host, port, username, auth,
-                        kerberos_service_name, password=None):
+def build_ssl_transport(host, port, username, auth, kerberos_service_name, password=None):
     transport = None
     if port is None:
         port = 10000
     if auth is None:
-        auth = 'NONE'
+        auth = "NONE"
     socket = TSSLSocket(host, port, cert_reqs=ssl.CERT_NONE)
-    if auth == 'NOSASL':
+    if auth == "NOSASL":
         # NOSASL corresponds to hive.server2.authentication=NOSASL
         # in hive-site.xml
         transport = thrift.transport.TTransport.TBufferedTransport(socket)
-    elif auth in ('LDAP', 'KERBEROS', 'NONE', 'CUSTOM'):
+    elif auth in ("LDAP", "KERBEROS", "NONE", "CUSTOM"):
         # Defer import so package dependency is optional
-        if auth == 'KERBEROS':
+        if auth == "KERBEROS":
             # KERBEROS mode in hive.server2.authentication is GSSAPI
             # in sasl library
-            sasl_auth = 'GSSAPI'
+            sasl_auth = "GSSAPI"
         else:
-            sasl_auth = 'PLAIN'
+            sasl_auth = "PLAIN"
             if password is None:
                 # Password doesn't matter in NONE mode, just needs
                 # to be nonempty.
-                password = 'x'
+                password = "x"
 
         def sasl_factory():
             sasl_client = sasl.Client()
-            sasl_client.setAttr('host', host)
-            if sasl_auth == 'GSSAPI':
-                sasl_client.setAttr('service', kerberos_service_name)
-            elif sasl_auth == 'PLAIN':
-                sasl_client.setAttr('username', username)
-                sasl_client.setAttr('password', password)
+            sasl_client.setAttr("host", host)
+            if sasl_auth == "GSSAPI":
+                sasl_client.setAttr("service", kerberos_service_name)
+            elif sasl_auth == "PLAIN":
+                sasl_client.setAttr("username", username)
+                sasl_client.setAttr("password", password)
             else:
                 raise AssertionError
             sasl_client.init()
             return sasl_client
 
-        transport = thrift_sasl.TSaslClientTransport(sasl_factory,
-                                                     sasl_auth, socket)
+        transport = thrift_sasl.TSaslClientTransport(sasl_factory, sasl_auth, socket)
     return transport
 
 
-def _is_retryable_error(exc: Exception) -> Optional[str]:
-    message = getattr(exc, 'message', None)
-    if message is None:
-        return None
-    message = message.lower()
-    if 'pending' in message:
-        return exc.message
-    if 'temporarily_unavailable' in message:
-        return exc.message
-    return None
+def _is_retryable_error(exc: Exception) -> str:
```

Comment:
Nice refactor!

For places like this where it's a refactor and not just formatting, can you call those out in the comments to make reviewing easier? Then it's easy to see what needs a closer look vs some newlines or spacing